scripts/SDM/julia_sdms/loadCHELSA.yml

```diff
@@ -0,0 +1,27 @@
+script: loadCHELSA.jl
```

Comment:
I'm not sure why you can't use CHELSA from the STAC, using loadFromStac?
https://coleo.biodiversite-quebec.ca/apps/io-layers/chelsa-clim/bio16
Do you need this to be in a specific format afterwards?

Reply:
no longer exists